##########################################################################
Using simulator FBL.
##########################################################################
=========================================
FBL [04 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7209 s: Imported packages\n2.5996 s: Created context on Tesla K80\n0.1701 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fbc947803c8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 04] completed in 4.19 s
=========================================
FBL [04 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7169 s: Imported packages\n2.8850 s: Created context on Tesla K80\n0.1657 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f278702f358>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 08] completed in 4.47 s
=========================================
FBL [04 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7357 s: Imported packages\n2.8959 s: Created context on Tesla K80\n0.1694 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fc374dec3c8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 12] completed in 4.49 s
=========================================
FBL [04 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7133 s: Imported packages\n2.5735 s: Created context on Tesla K80\n0.1623 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f0d5d155400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 16] completed in 4.13 s
=========================================
FBL [04 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7029 s: Imported packages\n2.9126 s: Created context on Tesla K80\n0.1722 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f13348f03c8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 24] completed in 4.47 s
=========================================
FBL [04 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7072 s: Imported packages\n2.8983 s: Created context on Tesla K80\n0.1684 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fa6f8967390>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [04 x 32] completed in 4.49 s
=========================================
FBL [08 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7211 s: Imported packages\n2.9014 s: Created context on Tesla K80\n0.1661 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f7e56928400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 04] completed in 4.50 s
=========================================
FBL [08 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7306 s: Imported packages\n2.5848 s: Created context on Tesla K80\n0.1660 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fb4822a9390>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 08] completed in 4.16 s
=========================================
FBL [08 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7196 s: Imported packages\n2.8940 s: Created context on Tesla K80\n0.1638 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f9796ed9400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 12] completed in 4.47 s
=========================================
FBL [08 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7213 s: Imported packages\n2.5949 s: Created context on Tesla K80\n0.1700 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f22128e53c8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 16] completed in 4.17 s
=========================================
FBL [08 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7052 s: Imported packages\n2.5780 s: Created context on Tesla K80\n0.1645 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fbadc808400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 24] completed in 4.11 s
=========================================
FBL [08 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7269 s: Imported packages\n2.8870 s: Created context on Tesla K80\n0.1726 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f815f26c400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [08 x 32] completed in 4.51 s
=========================================
FBL [12 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7175 s: Imported packages\n2.5986 s: Created context on Tesla K80\n0.1681 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f11f6303400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 04] completed in 4.16 s
=========================================
FBL [12 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7027 s: Imported packages\n2.5821 s: Created context on Tesla K80\n0.1635 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f08a51ce400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 08] completed in 4.15 s
=========================================
FBL [12 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7150 s: Imported packages\n2.5855 s: Created context on Tesla K80\n0.1718 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7faacc76c438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 12] completed in 4.21 s
=========================================
FBL [12 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7254 s: Imported packages\n2.5837 s: Created context on Tesla K80\n0.1655 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f24fccc3438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 16] completed in 4.17 s
=========================================
FBL [12 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7470 s: Imported packages\n2.8708 s: Created context on Tesla K80\n0.1732 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f4f95aa8438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 24] completed in 4.48 s
=========================================
FBL [12 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7210 s: Imported packages\n2.5821 s: Created context on Tesla K80\n0.1698 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f4658cc73c8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [12 x 32] completed in 4.16 s
=========================================
FBL [16 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7181 s: Imported packages\n2.5799 s: Created context on Tesla K80\n0.1618 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f9f2174c400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 04] completed in 4.13 s
=========================================
FBL [16 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7050 s: Imported packages\n2.5675 s: Created context on Tesla K80\n0.1646 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f85c3eb8400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 08] completed in 4.12 s
=========================================
FBL [16 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7076 s: Imported packages\n2.7231 s: Created context on Tesla K80\n0.1651 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f2d5bf2b400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 12] completed in 4.28 s
=========================================
FBL [16 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7237 s: Imported packages\n2.5703 s: Created context on Tesla K80\n0.1639 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f6aff6d3438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 16] completed in 4.15 s
=========================================
FBL [16 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7188 s: Imported packages\n2.5816 s: Created context on Tesla K80\n0.1632 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f512d11f470>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 24] completed in 4.17 s
=========================================
FBL [16 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7219 s: Imported packages\n2.8920 s: Created context on Tesla K80\n0.1640 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f4e65830438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [16 x 32] completed in 4.51 s
=========================================
FBL [24 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7260 s: Imported packages\n2.6035 s: Created context on Tesla K80\n0.1665 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f41c06622e8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 04] completed in 4.17 s
=========================================
FBL [24 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7079 s: Imported packages\n2.8736 s: Created context on Tesla K80\n0.1784 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fe6efd592e8>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 08] completed in 4.49 s
=========================================
FBL [24 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7250 s: Imported packages\n2.5944 s: Created context on Tesla K80\n0.1772 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f67a5bac470>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 12] completed in 4.19 s
=========================================
FBL [24 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.6990 s: Imported packages\n2.7330 s: Created context on Tesla K80\n0.1705 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f8d536a5470>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 16] completed in 4.32 s
=========================================
FBL [24 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7689 s: Imported packages\n2.5932 s: Created context on Tesla K80\n0.1718 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f83d91cc438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 24] completed in 4.21 s
=========================================
FBL [24 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7147 s: Imported packages\n2.9403 s: Created context on Tesla K80\n0.1849 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f41dec54438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [24 x 32] completed in 4.56 s
=========================================
FBL [32 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7427 s: Imported packages\n2.5992 s: Created context on Tesla K80\n0.1712 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f8989e1e400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 04] completed in 4.25 s
=========================================
FBL [32 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7347 s: Imported packages\n2.5708 s: Created context on Tesla K80\n0.1685 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7fbfaca1a470>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 08] completed in 4.15 s
=========================================
FBL [32 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7071 s: Imported packages\n2.8969 s: Created context on Tesla K80\n0.1678 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f08ebdf2400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 12] completed in 4.46 s
=========================================
FBL [32 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7018 s: Imported packages\n2.6024 s: Created context on Tesla K80\n0.1726 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f5fbf0b4470>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 16] completed in 4.18 s
=========================================
FBL [32 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7306 s: Imported packages\n2.8849 s: Created context on Tesla K80\n0.1708 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f55a8920400>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 24] completed in 4.52 s
=========================================
FBL [32 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py:125: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/FBL_step_kernel.cu(190): warning: variable "V_row" was declared but never referenced\n\n\n  jit_compile_args={\n0.7477 s: Imported packages\n2.5877 s: Created context on Tesla K80\n0.1707 s: Generated initial conditions\nTraceback (most recent call last):\n  File "run_benchmark.py", line 256, in <module>\n    sim = initFBL()\n  File "run_benchmark.py", line 200, in initFBL\n    **kwargs)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 139, in __init__\n    self.H = Common.CUDAArray2D(self.gpu_stream, nx, ny, ghost_cells_x, ghost_cells_y, H)\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Common.py", line 322, in __init__\n    assert(host_data.shape[1] == self.nx_halo), str(host_data.shape[1]) + " vs " + str(self.nx_halo)\nAssertionError: 2048 vs 2050\nException ignored in: <bound method Simulator.__del__ of <SWESimulators.FBL.FBL object at 0x7f0ee340c438>>\nTraceback (most recent call last):\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/Simulator.py", line 228, in __del__\n    self.cleanUp()\n  File "/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/FBL.py", line 235, in cleanUp\n    self.gpu_data.release()\nAttributeError: \'FBL\' object has no attribute \'gpu_data\'\n'
=========================================

FBL [32 x 32] completed in 4.21 s



##########################################################################
Using simulator CTCS.
##########################################################################
=========================================
CTCS [04 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:373: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  self.boundaryKernels = gpu_ctx.get_kernel("CTCS_boundary.cu", defines={\'block_width\': block_width, \'block_height\': block_height})\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:374: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  self.boundaryKernels_NS = gpu_ctx.get_kernel("CTCS_boundary_NS.cu", defines={\'block_width\': self.local_size_NS[0], \'block_height\': self.local_size_NS[1]})\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:375: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  self.boundaryKernels_EW = gpu_ctx.get_kernel("CTCS_boundary_EW.cu", defines={\'block_width\': self.local_size_EW[0], \'block_height\': self.local_size_EW[1]})\n0.7330 s: Imported packages\n2.8970 s: Created context on Tesla K80\n0.1662 s: Generated initial conditions\n1.4999 s: Created CTCS simulator\n0.1506 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 4] ===\n100 %\n `-> 3.7683 s: Step, 333.9158 mcells/sec\n `-> 0.0357 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 333.91580365 ===\n'
=========================================

CTCS [04 x 04] completed in 9.46 s
=========================================
CTCS [04 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7167 s: Imported packages\n2.5850 s: Created context on Tesla K80\n0.1567 s: Generated initial conditions\n0.4813 s: Created CTCS simulator\n0.0839 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 8] ===\n100 %\n `-> 2.1564 s: Step, 583.5057 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 583.50572419 ===\n'
=========================================

CTCS [04 x 08] completed in 6.46 s
=========================================
CTCS [04 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7310 s: Imported packages\n2.8647 s: Created context on Tesla K80\n0.1651 s: Generated initial conditions\n0.4947 s: Created CTCS simulator\n0.0897 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 12] ===\n100 %\n `-> 1.8493 s: Step, 680.4191 mcells/sec\n `-> 0.0359 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 680.41910431 ===\n'
=========================================

CTCS [04 x 12] completed in 6.45 s
=========================================
CTCS [04 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6991 s: Imported packages\n2.9006 s: Created context on Tesla K80\n0.1575 s: Generated initial conditions\n0.4936 s: Created CTCS simulator\n0.0732 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 16] ===\n100 %\n `-> 1.3986 s: Step, 899.6619 mcells/sec\n `-> 0.0364 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 899.66186554 ===\n'
=========================================

CTCS [04 x 16] completed in 5.97 s
=========================================
CTCS [04 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7119 s: Imported packages\n2.5837 s: Created context on Tesla K80\n0.1607 s: Generated initial conditions\n0.4870 s: Created CTCS simulator\n0.0615 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 24] ===\n100 %\n `-> 1.1937 s: Step, 1054.1058 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1054.10580080 ===\n'
=========================================

CTCS [04 x 24] completed in 5.48 s
=========================================
CTCS [04 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7369 s: Imported packages\n2.5981 s: Created context on Tesla K80\n0.1640 s: Generated initial conditions\n0.4811 s: Created CTCS simulator\n0.0603 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 32] ===\n100 %\n `-> 1.1862 s: Step, 1060.7351 mcells/sec\n `-> 0.0359 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1060.73512921 ===\n'
=========================================

CTCS [04 x 32] completed in 5.47 s
=========================================
CTCS [08 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7195 s: Imported packages\n2.5878 s: Created context on Tesla K80\n0.1586 s: Generated initial conditions\n0.4892 s: Created CTCS simulator\n0.0800 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 4] ===\n100 %\n `-> 1.9683 s: Step, 639.2627 mcells/sec\n `-> 0.0371 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 639.26271575 ===\n'
=========================================

CTCS [08 x 04] completed in 6.30 s
=========================================
CTCS [08 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7359 s: Imported packages\n2.9024 s: Created context on Tesla K80\n0.1686 s: Generated initial conditions\n0.4920 s: Created CTCS simulator\n0.0655 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 8] ===\n100 %\n `-> 1.1658 s: Step, 1079.3759 mcells/sec\n `-> 0.0358 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1079.37587090 ===\n'
=========================================

CTCS [08 x 08] completed in 5.81 s
=========================================
CTCS [08 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7201 s: Imported packages\n2.8871 s: Created context on Tesla K80\n0.1748 s: Generated initial conditions\n0.5026 s: Created CTCS simulator\n0.0597 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 12] ===\n100 %\n `-> 0.9408 s: Step, 1337.4366 mcells/sec\n `-> 0.0360 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1337.43657996 ===\n'
=========================================

CTCS [08 x 12] completed in 5.54 s
=========================================
CTCS [08 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7137 s: Imported packages\n2.5978 s: Created context on Tesla K80\n0.1595 s: Generated initial conditions\n0.4782 s: Created CTCS simulator\n0.0514 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 16] ===\n100 %\n `-> 0.8366 s: Step, 1504.1070 mcells/sec\n `-> 0.0357 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1504.10701383 ===\n'
=========================================

CTCS [08 x 16] completed in 5.09 s
=========================================
CTCS [08 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7237 s: Imported packages\n2.8966 s: Created context on Tesla K80\n0.1605 s: Generated initial conditions\n0.4830 s: Created CTCS simulator\n0.0589 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 24] ===\n100 %\n `-> 0.9203 s: Step, 1367.3140 mcells/sec\n `-> 0.0360 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1367.31404063 ===\n'
=========================================

CTCS [08 x 24] completed in 5.50 s
=========================================
CTCS [08 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7030 s: Imported packages\n2.5643 s: Created context on Tesla K80\n0.1686 s: Generated initial conditions\n0.4970 s: Created CTCS simulator\n0.0531 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 32] ===\n100 %\n `-> 0.8499 s: Step, 1480.5338 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1480.53378248 ===\n'
=========================================

CTCS [08 x 32] completed in 5.09 s
=========================================
CTCS [12 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7241 s: Imported packages\n2.8916 s: Created context on Tesla K80\n0.1714 s: Generated initial conditions\n0.4940 s: Created CTCS simulator\n0.0771 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 4] ===\n100 %\n `-> 1.4617 s: Step, 860.8567 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 860.85668710 ===\n'
=========================================

CTCS [12 x 04] completed in 6.08 s
=========================================
CTCS [12 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7044 s: Imported packages\n2.5854 s: Created context on Tesla K80\n0.1579 s: Generated initial conditions\n0.4781 s: Created CTCS simulator\n0.0520 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 8] ===\n100 %\n `-> 0.8327 s: Step, 1511.0161 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1511.01608095 ===\n'
=========================================

CTCS [12 x 08] completed in 5.07 s
=========================================
CTCS [12 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7238 s: Imported packages\n2.8922 s: Created context on Tesla K80\n0.1581 s: Generated initial conditions\n0.4826 s: Created CTCS simulator\n0.0578 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 12] ===\n100 %\n `-> 0.8647 s: Step, 1455.1568 mcells/sec\n `-> 0.0364 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1455.15680029 ===\n'
=========================================

CTCS [12 x 12] completed in 5.47 s
=========================================
CTCS [12 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7017 s: Imported packages\n2.5907 s: Created context on Tesla K80\n0.1659 s: Generated initial conditions\n0.4924 s: Created CTCS simulator\n0.0516 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 16] ===\n100 %\n `-> 0.7784 s: Step, 1616.5708 mcells/sec\n `-> 0.0366 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1616.57078206 ===\n'
=========================================

CTCS [12 x 16] completed in 5.06 s
=========================================
CTCS [12 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7164 s: Imported packages\n2.8800 s: Created context on Tesla K80\n0.1678 s: Generated initial conditions\n0.4978 s: Created CTCS simulator\n0.0574 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 24] ===\n100 %\n `-> 0.8494 s: Step, 1481.4198 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1481.41980059 ===\n'
=========================================

CTCS [12 x 24] completed in 5.45 s
=========================================
CTCS [12 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7147 s: Imported packages\n2.5842 s: Created context on Tesla K80\n0.1642 s: Generated initial conditions\n0.4823 s: Created CTCS simulator\n0.0526 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 32] ===\n100 %\n `-> 0.8223 s: Step, 1530.1860 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1530.18597794 ===\n'
=========================================

CTCS [12 x 32] completed in 5.09 s
=========================================
CTCS [16 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7121 s: Imported packages\n2.8647 s: Created context on Tesla K80\n0.1663 s: Generated initial conditions\n0.4884 s: Created CTCS simulator\n0.0659 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 4] ===\n100 %\n `-> 1.0879 s: Step, 1156.5951 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1156.59513995 ===\n'
=========================================

CTCS [16 x 04] completed in 5.64 s
=========================================
CTCS [16 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7266 s: Imported packages\n2.5924 s: Created context on Tesla K80\n0.1617 s: Generated initial conditions\n0.4906 s: Created CTCS simulator\n0.0482 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 8] ===\n100 %\n `-> 0.6868 s: Step, 1832.2178 mcells/sec\n `-> 0.0358 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1832.21782711 ===\n'
=========================================

CTCS [16 x 08] completed in 4.96 s
=========================================
CTCS [16 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7306 s: Imported packages\n2.8969 s: Created context on Tesla K80\n0.1601 s: Generated initial conditions\n0.4952 s: Created CTCS simulator\n0.0522 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 12] ===\n100 %\n `-> 0.7300 s: Step, 1723.6004 mcells/sec\n `-> 0.0358 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1723.60040213 ===\n'
=========================================

CTCS [16 x 12] completed in 5.33 s
=========================================
CTCS [16 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7369 s: Imported packages\n2.8768 s: Created context on Tesla K80\n0.1599 s: Generated initial conditions\n0.4950 s: Created CTCS simulator\n0.0544 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 16] ===\n100 %\n `-> 0.7594 s: Step, 1656.9968 mcells/sec\n `-> 0.0364 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1656.99681399 ===\n'
=========================================

CTCS [16 x 16] completed in 5.36 s
=========================================
CTCS [16 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7141 s: Imported packages\n2.5781 s: Created context on Tesla K80\n0.1580 s: Generated initial conditions\n0.4773 s: Created CTCS simulator\n0.0511 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 24] ===\n100 %\n `-> 0.7576 s: Step, 1660.8848 mcells/sec\n `-> 0.0366 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1660.88481190 ===\n'
=========================================

CTCS [16 x 24] completed in 5.02 s
=========================================
CTCS [16 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7304 s: Imported packages\n2.5614 s: Created context on Tesla K80\n0.1649 s: Generated initial conditions\n0.4939 s: Created CTCS simulator\n0.0525 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 32] ===\n100 %\n `-> 0.7706 s: Step, 1632.9118 mcells/sec\n `-> 0.0365 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1632.91178896 ===\n'
=========================================

CTCS [16 x 32] completed in 5.04 s
=========================================
CTCS [24 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7037 s: Imported packages\n2.8796 s: Created context on Tesla K80\n0.1612 s: Generated initial conditions\n0.4904 s: Created CTCS simulator\n0.0579 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 4] ===\n100 %\n `-> 0.8434 s: Step, 1491.8409 mcells/sec\n `-> 0.0372 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1491.84092776 ===\n'
=========================================

CTCS [24 x 04] completed in 5.40 s
=========================================
CTCS [24 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7097 s: Imported packages\n2.8982 s: Created context on Tesla K80\n0.1597 s: Generated initial conditions\n0.4906 s: Created CTCS simulator\n0.0541 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 8] ===\n100 %\n `-> 0.7622 s: Step, 1650.8517 mcells/sec\n `-> 0.0355 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1650.85174345 ===\n'
=========================================

CTCS [24 x 08] completed in 5.33 s
=========================================
CTCS [24 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6981 s: Imported packages\n2.5933 s: Created context on Tesla K80\n0.1615 s: Generated initial conditions\n0.4910 s: Created CTCS simulator\n0.0495 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 12] ===\n100 %\n `-> 0.6969 s: Step, 1805.6386 mcells/sec\n `-> 0.0360 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1805.63856249 ===\n'
=========================================

CTCS [24 x 12] completed in 4.95 s
=========================================
CTCS [24 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7053 s: Imported packages\n2.9106 s: Created context on Tesla K80\n0.1680 s: Generated initial conditions\n0.4972 s: Created CTCS simulator\n0.0558 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 16] ===\n100 %\n `-> 0.7676 s: Step, 1639.3489 mcells/sec\n `-> 0.0364 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1639.34886871 ===\n'
=========================================

CTCS [24 x 16] completed in 5.37 s
=========================================
CTCS [24 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7032 s: Imported packages\n2.5864 s: Created context on Tesla K80\n0.1612 s: Generated initial conditions\n0.4822 s: Created CTCS simulator\n0.0532 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 24] ===\n100 %\n `-> 0.8279 s: Step, 1519.9278 mcells/sec\n `-> 0.0368 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1519.92776368 ===\n'
=========================================

CTCS [24 x 24] completed in 5.09 s
=========================================
CTCS [24 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7192 s: Imported packages\n2.5930 s: Created context on Tesla K80\n0.1646 s: Generated initial conditions\n0.5020 s: Created CTCS simulator\n0.0557 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 32] ===\n100 %\n `-> 0.9381 s: Step, 1341.3357 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1341.33573864 ===\n'
=========================================

CTCS [24 x 32] completed in 5.28 s
=========================================
CTCS [32 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7550 s: Imported packages\n2.5902 s: Created context on Tesla K80\n0.1594 s: Generated initial conditions\n0.4849 s: Created CTCS simulator\n0.0468 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 4] ===\n100 %\n `-> 0.6157 s: Step, 2043.5508 mcells/sec\n `-> 0.0357 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 2043.55076484 ===\n'
=========================================

CTCS [32 x 04] completed in 4.92 s
=========================================
CTCS [32 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7086 s: Imported packages\n2.5851 s: Created context on Tesla K80\n0.1634 s: Generated initial conditions\n0.4862 s: Created CTCS simulator\n0.0475 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 8] ===\n100 %\n `-> 0.6097 s: Step, 2063.9046 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 2063.90458231 ===\n'
=========================================

CTCS [32 x 08] completed in 4.87 s
=========================================
CTCS [32 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7201 s: Imported packages\n2.5901 s: Created context on Tesla K80\n0.1597 s: Generated initial conditions\n0.4929 s: Created CTCS simulator\n0.0486 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 12] ===\n100 %\n `-> 0.6407 s: Step, 1963.7891 mcells/sec\n `-> 0.0366 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1963.78913725 ===\n'
=========================================

CTCS [32 x 12] completed in 4.92 s
=========================================
CTCS [32 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7030 s: Imported packages\n2.7099 s: Created context on Tesla K80\n0.1624 s: Generated initial conditions\n0.4857 s: Created CTCS simulator\n0.0510 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 16] ===\n100 %\n `-> 0.6593 s: Step, 1908.4464 mcells/sec\n `-> 0.0366 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1908.44638909 ===\n'
=========================================

CTCS [32 x 16] completed in 5.04 s
=========================================
CTCS [32 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7083 s: Imported packages\n2.5704 s: Created context on Tesla K80\n0.1652 s: Generated initial conditions\n0.4824 s: Created CTCS simulator\n0.0529 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 24] ===\n100 %\n `-> 0.8487 s: Step, 1482.6954 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1482.69541526 ===\n'
=========================================

CTCS [32 x 24] completed in 5.13 s
=========================================
CTCS [32 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CTCS.py:134: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7082 s: Imported packages\n2.6035 s: Created context on Tesla K80\n0.1628 s: Generated initial conditions\n0.4763 s: Created CTCS simulator\n0.0508 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 32] ===\n100 %\n `-> 0.7467 s: Step, 1685.0678 mcells/sec\n `-> 0.0359 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 1685.06775938 ===\n'
=========================================

CTCS [32 x 32] completed in 5.00 s



##########################################################################
Using simulator CDKLM.
##########################################################################
=========================================
CDKLM [04 x 04]
-----------------------------------------
b'0.7265 s: Imported packages\n2.6048 s: Created context on Tesla K80\n0.1772 s: Generated initial conditions\n0.0706 s: Created CDKLM simulator\n0.1990 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 4] ===\n100 %\n `-> 10.0806 s: Step, 124.8233 mcells/sec\n `-> 0.0365 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 124.82329985 ===\n'
=========================================

CDKLM [04 x 04] completed in 14.13 s
=========================================
CDKLM [04 x 08]
-----------------------------------------
b'0.7355 s: Imported packages\n2.5755 s: Created context on Tesla K80\n0.1747 s: Generated initial conditions\n0.0676 s: Created CDKLM simulator\n0.1354 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 8] ===\n100 %\n `-> 6.3611 s: Step, 197.8109 mcells/sec\n `-> 0.0360 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 197.81093990 ===\n'
=========================================

CDKLM [04 x 08] completed in 10.32 s
=========================================
CDKLM [04 x 12]
-----------------------------------------
b'0.7048 s: Imported packages\n2.8906 s: Created context on Tesla K80\n0.1668 s: Generated initial conditions\n0.0657 s: Created CDKLM simulator\n0.1377 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 12] ===\n100 %\n `-> 5.5747 s: Step, 225.7135 mcells/sec\n `-> 0.0359 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 225.71347895 ===\n'
=========================================

CDKLM [04 x 12] completed in 9.79 s
=========================================
CDKLM [04 x 16]
-----------------------------------------
b'0.7344 s: Imported packages\n2.8994 s: Created context on Tesla K80\n0.1750 s: Generated initial conditions\n0.0679 s: Created CDKLM simulator\n0.1182 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 16] ===\n100 %\n `-> 4.3959 s: Step, 286.2434 mcells/sec\n `-> 0.0363 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 286.24336369 ===\n'
=========================================

CDKLM [04 x 16] completed in 8.65 s
=========================================
CDKLM [04 x 24]
-----------------------------------------
b'0.7357 s: Imported packages\n2.8908 s: Created context on Tesla K80\n0.1703 s: Generated initial conditions\n0.0680 s: Created CDKLM simulator\n0.1058 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 24] ===\n100 %\n `-> 3.8952 s: Step, 323.0373 mcells/sec\n `-> 0.0365 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 323.03730035 ===\n'
=========================================

CDKLM [04 x 24] completed in 8.12 s
=========================================
CDKLM [04 x 32]
-----------------------------------------
b'0.6996 s: Imported packages\n2.5913 s: Created context on Tesla K80\n0.1682 s: Generated initial conditions\n0.0643 s: Created CDKLM simulator\n0.0800 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [4 x 32] ===\n100 %\n `-> 3.7138 s: Step, 338.8144 mcells/sec\n `-> 0.0359 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 338.81443123 ===\n'
=========================================

CDKLM [04 x 32] completed in 7.57 s
=========================================
CDKLM [08 x 04]
-----------------------------------------
b'0.7031 s: Imported packages\n2.5769 s: Created context on Tesla K80\n0.1654 s: Generated initial conditions\n0.0649 s: Created CDKLM simulator\n0.1209 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 4] ===\n100 %\n `-> 5.7086 s: Step, 220.4186 mcells/sec\n `-> 0.0358 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 220.41857341 ===\n'
=========================================

CDKLM [08 x 04] completed in 9.61 s
=========================================
CDKLM [08 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6945 s: Imported packages\n2.5974 s: Created context on Tesla K80\n0.1683 s: Generated initial conditions\n0.5865 s: Created CDKLM simulator\n0.0885 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 8] ===\n100 %\n `-> 3.7108 s: Step, 339.0903 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 339.09033012 ===\n'
=========================================

CDKLM [08 x 08] completed in 8.13 s
=========================================
CDKLM [08 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7299 s: Imported packages\n2.5776 s: Created context on Tesla K80\n0.1691 s: Generated initial conditions\n0.5855 s: Created CDKLM simulator\n0.0785 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 12] ===\n100 %\n `-> 3.0139 s: Step, 417.4912 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 417.49120730 ===\n'
=========================================

CDKLM [08 x 12] completed in 7.42 s
=========================================
CDKLM [08 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7045 s: Imported packages\n2.8912 s: Created context on Tesla K80\n0.1680 s: Generated initial conditions\n0.5969 s: Created CDKLM simulator\n0.0931 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 16] ===\n100 %\n `-> 2.9488 s: Step, 426.7154 mcells/sec\n `-> 0.0372 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 426.71537048 ===\n'
=========================================

CDKLM [08 x 16] completed in 7.69 s
=========================================
CDKLM [08 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7170 s: Imported packages\n2.5841 s: Created context on Tesla K80\n0.1637 s: Generated initial conditions\n0.5798 s: Created CDKLM simulator\n0.0733 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 24] ===\n100 %\n `-> 2.9495 s: Step, 426.6184 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 426.61837486 ===\n'
=========================================

CDKLM [08 x 24] completed in 7.33 s
=========================================
CDKLM [08 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7021 s: Imported packages\n2.5961 s: Created context on Tesla K80\n0.1729 s: Generated initial conditions\n0.5808 s: Created CDKLM simulator\n0.0731 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [8 x 32] ===\n100 %\n `-> 2.8540 s: Step, 440.8930 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 440.89295864 ===\n'
=========================================

CDKLM [08 x 32] completed in 7.25 s
=========================================
CDKLM [12 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6977 s: Imported packages\n2.5767 s: Created context on Tesla K80\n0.1645 s: Generated initial conditions\n0.5764 s: Created CDKLM simulator\n0.1075 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 4] ===\n100 %\n `-> 4.5535 s: Step, 276.3380 mcells/sec\n `-> 0.0363 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 276.33797896 ===\n'
=========================================

CDKLM [12 x 04] completed in 8.96 s
=========================================
CDKLM [12 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7183 s: Imported packages\n2.9724 s: Created context on Tesla K80\n0.1703 s: Generated initial conditions\n0.5835 s: Created CDKLM simulator\n0.0966 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 8] ===\n100 %\n `-> 3.0673 s: Step, 410.2279 mcells/sec\n `-> 0.0368 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 410.22790302 ===\n'
=========================================

CDKLM [12 x 08] completed in 7.88 s
=========================================
CDKLM [12 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7343 s: Imported packages\n2.5803 s: Created context on Tesla K80\n0.1647 s: Generated initial conditions\n0.5774 s: Created CDKLM simulator\n0.0727 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 12] ===\n100 %\n `-> 2.9723 s: Step, 423.3322 mcells/sec\n `-> 0.0361 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 423.33222961 ===\n'
=========================================

CDKLM [12 x 12] completed in 7.36 s
=========================================
CDKLM [12 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7235 s: Imported packages\n2.8876 s: Created context on Tesla K80\n0.1641 s: Generated initial conditions\n0.5914 s: Created CDKLM simulator\n0.0906 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 16] ===\n100 %\n `-> 2.8810 s: Step, 436.7606 mcells/sec\n `-> 0.0371 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 436.76057549 ===\n'
=========================================

CDKLM [12 x 16] completed in 7.60 s
=========================================
CDKLM [12 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7183 s: Imported packages\n2.5996 s: Created context on Tesla K80\n0.1752 s: Generated initial conditions\n0.5936 s: Created CDKLM simulator\n0.0759 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 24] ===\n100 %\n `-> 2.8128 s: Step, 447.3418 mcells/sec\n `-> 0.0360 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 447.34180128 ===\n'
=========================================

CDKLM [12 x 24] completed in 7.26 s
=========================================
CDKLM [12 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7141 s: Imported packages\n2.9034 s: Created context on Tesla K80\n0.1683 s: Generated initial conditions\n0.5937 s: Created CDKLM simulator\n0.0837 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [12 x 32] ===\n100 %\n `-> 2.9101 s: Step, 432.3821 mcells/sec\n `-> 0.0375 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 432.38205644 ===\n'
=========================================

CDKLM [12 x 32] completed in 7.69 s
=========================================
CDKLM [16 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7044 s: Imported packages\n2.5924 s: Created context on Tesla K80\n0.1693 s: Generated initial conditions\n0.5766 s: Created CDKLM simulator\n0.0842 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 4] ===\n100 %\n `-> 3.4520 s: Step, 364.5160 mcells/sec\n `-> 0.0368 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 364.51602323 ===\n'
=========================================

CDKLM [16 x 04] completed in 7.88 s
=========================================
CDKLM [16 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7504 s: Imported packages\n2.9387 s: Created context on Tesla K80\n0.1775 s: Generated initial conditions\n0.6042 s: Created CDKLM simulator\n0.0778 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 8] ===\n100 %\n `-> 2.6639 s: Step, 472.3408 mcells/sec\n `-> 0.0367 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 472.34082547 ===\n'
=========================================

CDKLM [16 x 08] completed in 7.50 s
=========================================
CDKLM [16 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6957 s: Imported packages\n2.6134 s: Created context on Tesla K80\n0.1678 s: Generated initial conditions\n0.5916 s: Created CDKLM simulator\n0.0712 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 12] ===\n100 %\n `-> 2.6426 s: Step, 476.1538 mcells/sec\n `-> 0.0365 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 476.15375242 ===\n'
=========================================

CDKLM [16 x 12] completed in 7.07 s
=========================================
CDKLM [16 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7172 s: Imported packages\n2.5831 s: Created context on Tesla K80\n0.1698 s: Generated initial conditions\n0.5755 s: Created CDKLM simulator\n0.0673 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 16] ===\n100 %\n `-> 2.5697 s: Step, 489.6683 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 489.66826414 ===\n'
=========================================

CDKLM [16 x 16] completed in 6.94 s
=========================================
CDKLM [16 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7044 s: Imported packages\n2.5800 s: Created context on Tesla K80\n0.1646 s: Generated initial conditions\n0.5823 s: Created CDKLM simulator\n0.0722 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 24] ===\n100 %\n `-> 2.6915 s: Step, 467.5002 mcells/sec\n `-> 0.0371 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 467.50017989 ===\n'
=========================================

CDKLM [16 x 24] completed in 7.06 s
=========================================
CDKLM [16 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6972 s: Imported packages\n2.5739 s: Created context on Tesla K80\n0.1745 s: Generated initial conditions\n0.5809 s: Created CDKLM simulator\n0.0706 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [16 x 32] ===\n100 %\n `-> 2.6935 s: Step, 467.1585 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 467.15853519 ===\n'
=========================================

CDKLM [16 x 32] completed in 7.05 s
=========================================
CDKLM [24 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7184 s: Imported packages\n2.8957 s: Created context on Tesla K80\n0.1664 s: Generated initial conditions\n0.5885 s: Created CDKLM simulator\n0.0855 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 4] ===\n100 %\n `-> 3.0205 s: Step, 416.5785 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 416.57852380 ===\n'
=========================================

CDKLM [24 x 04] completed in 7.76 s
=========================================
CDKLM [24 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7102 s: Imported packages\n2.5809 s: Created context on Tesla K80\n0.1629 s: Generated initial conditions\n0.5714 s: Created CDKLM simulator\n0.0729 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 8] ===\n100 %\n `-> 2.7538 s: Step, 456.9283 mcells/sec\n `-> 0.0364 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 456.92830592 ===\n'
=========================================

CDKLM [24 x 08] completed in 7.12 s
=========================================
CDKLM [24 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7247 s: Imported packages\n2.5871 s: Created context on Tesla K80\n0.1681 s: Generated initial conditions\n0.5819 s: Created CDKLM simulator\n0.0709 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 12] ===\n100 %\n `-> 2.6409 s: Step, 476.4593 mcells/sec\n `-> 0.0369 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 476.45925735 ===\n'
=========================================

CDKLM [24 x 12] completed in 7.06 s
=========================================
CDKLM [24 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7228 s: Imported packages\n2.8962 s: Created context on Tesla K80\n0.1696 s: Generated initial conditions\n0.5891 s: Created CDKLM simulator\n0.0859 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 16] ===\n100 %\n `-> 2.7529 s: Step, 457.0845 mcells/sec\n `-> 0.0370 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 457.08446259 ===\n'
=========================================

CDKLM [24 x 16] completed in 7.48 s
=========================================
CDKLM [24 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7189 s: Imported packages\n2.6078 s: Created context on Tesla K80\n0.1628 s: Generated initial conditions\n0.5873 s: Created CDKLM simulator\n0.0745 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 24] ===\n100 %\n `-> 2.9220 s: Step, 430.6330 mcells/sec\n `-> 0.0362 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 430.63299372 ===\n'
=========================================

CDKLM [24 x 24] completed in 7.33 s
=========================================
CDKLM [24 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.6945 s: Imported packages\n2.8846 s: Created context on Tesla K80\n0.1715 s: Generated initial conditions\n0.5976 s: Created CDKLM simulator\n0.0998 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [24 x 32] ===\n100 %\n `-> 3.1884 s: Step, 394.6509 mcells/sec\n `-> 0.0370 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 394.65086923 ===\n'
=========================================

CDKLM [24 x 32] completed in 7.90 s
=========================================
CDKLM [32 x 04]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7255 s: Imported packages\n2.8935 s: Created context on Tesla K80\n0.1727 s: Generated initial conditions\n0.5970 s: Created CDKLM simulator\n0.0813 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 4] ===\n100 %\n `-> 2.5096 s: Step, 501.3978 mcells/sec\n `-> 0.0370 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 501.39777434 ===\n'
=========================================

CDKLM [32 x 04] completed in 7.27 s
=========================================
CDKLM [32 x 08]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7251 s: Imported packages\n2.8856 s: Created context on Tesla K80\n0.1623 s: Generated initial conditions\n0.5763 s: Created CDKLM simulator\n0.0745 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 8] ===\n100 %\n `-> 2.3275 s: Step, 540.6250 mcells/sec\n `-> 0.0389 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 540.62499560 ===\n'
=========================================

CDKLM [32 x 08] completed in 7.05 s
=========================================
CDKLM [32 x 12]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7338 s: Imported packages\n2.5990 s: Created context on Tesla K80\n0.1711 s: Generated initial conditions\n0.5965 s: Created CDKLM simulator\n0.0666 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 12] ===\n100 %\n `-> 2.2920 s: Step, 548.9905 mcells/sec\n `-> 0.0361 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 548.99048684 ===\n'
=========================================

CDKLM [32 x 12] completed in 6.72 s
=========================================
CDKLM [32 x 16]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7148 s: Imported packages\n2.5685 s: Created context on Tesla K80\n0.1710 s: Generated initial conditions\n0.5989 s: Created CDKLM simulator\n0.0640 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 16] ===\n100 %\n `-> 2.2787 s: Step, 552.2077 mcells/sec\n `-> 0.0370 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 552.20773126 ===\n'
=========================================

CDKLM [32 x 16] completed in 6.67 s
=========================================
CDKLM [32 x 24]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7185 s: Imported packages\n2.6016 s: Created context on Tesla K80\n0.1709 s: Generated initial conditions\n0.5948 s: Created CDKLM simulator\n0.0724 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 24] ===\n100 %\n `-> 2.6785 s: Step, 469.7751 mcells/sec\n `-> 0.0370 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 469.77508815 ===\n'
=========================================

CDKLM [32 x 24] completed in 7.10 s
=========================================
CDKLM [32 x 32]
-----------------------------------------
b'/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/CDKLM16.py:150: UserWarning: The CUDA compiler succeeded, but said the following:\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(583): warning: dynamic initialization in unreachable code\n\n/home/ubuntu/jupyter_notebooks/gpu-ocean/gpu_ocean/SWESimulators/gpu_kernels/common.cu(613): warning: dynamic initialization in unreachable code\n\n\n  jit_compile_args={\n0.7178 s: Imported packages\n2.5953 s: Created context on Tesla K80\n0.1693 s: Generated initial conditions\n0.5838 s: Created CDKLM simulator\n0.0695 s: Spinup of simulator\n=== Running with domain size [2048 x 2048], block size [32 x 32] ===\n100 %\n `-> 2.4585 s: Step, 511.8027 mcells/sec\n `-> 0.0363 s: Download\n `-> t_sim=2.8975, h_max=1.0000\n === Maximum megacells: 511.80266106 ===\n'
=========================================

CDKLM [32 x 32] completed in 6.87 s



